lib/Electron.js

```diff
@@ -34,10 +34,7 @@ Electron.prototype.start = function () {
 
   self.controller = setup_test_instance(self._opt, function (err, url) {
     if (err) {
-      self.emit('error', err)
-      self.emit('done', {
```

Comment:
Note that we're not emitting `'done'` anymore if there's an error. This is what `SauceBrowser.js` does.

Reply:
To make sure I understand: `SauceBrowser` also doesn't emit `done`?